cmd/build-images.go

```diff
@@ -31,13 +32,22 @@ Before running this command, you should run ` + "`fissile build layer stemcell`"
 The images will be tagged: ` + "`<repository>-<role_name>:<SIGNATURE>`" + `.
 The SIGNATURE is based on the hashes of all jobs and packages that are included in
 the image.
+
+The --patch-properties-release flag is used to distinguish the patchProperties release/job spec
+from other specs.  At most one is allowed.  Its syntax is --patch-properties-release=<RELEASE>/<JOB>.
+For hcf: the value should be "hcf/patch-properties".
```

Comment:
We don't need to document HCF here. (If we forget, we can look at `hcf.git`'s `.fissilerc` or something.)